core/templates/dev/head/pages/exploration_editor/statistics_tab/PieChartDirective.js

```diff
@@ -35,27 +35,31 @@ oppia.directive('pieChart', [function() {
       var chart = null;
 
       var redrawChart = function() {
-        if (!chart) {
-          chart = new google.visualization.PieChart($element[0]);
-        }
-        chart.draw(google.visualization.arrayToDataTable($scope.data()), {
-          title: options.title,
-          pieHole: options.pieHole,
-          pieSliceTextStyle: {
-            color: options.pieSliceTextStyleColor,
-          },
-          pieSliceBorderColor: options.pieSliceBorderColor,
-          pieSliceText: 'none',
-          chartArea: {
-            left: options.left,
-            width: options.chartAreaWidth
-          },
-          colors: options.colors,
-          height: options.height,
-          legend: {
-            position: options.legendPosition || 'none'
-          },
-          width: options.width
+        // Need to wait for load statement in editor template to finish.
+        // https://stackoverflow.com/questions/42714876/google-visualization-piechart-is-not-a-constructor
+        google.charts.setOnLoadCallback(function () {
```

Comment:
Just to check, do we need to do this every time we "redraw chart" (which is what it looks like now), or would doing it once at the outset suffice?

Reply:
So I think we can 
 - move the chart declaration outside of the function and just have the callback for the declaration.
 - redrawChart need to do a null check for `chart`
Does that sound good?

Reply:
Yes, if it works :)